### PR TITLE
Add special field to random resource, enhance example

### PR DESCRIPTION
**Pull request by lucasmelogithub**

### CODE REVIEW

File: examples/intel-optimized-mysql-server-replica_testing/outputs.tf [hunk 1]
```diff
@@ -73,4 +73,86 @@ output "mysql_maintenance_window" {
...
+  description = "MySQL instance root username"
+  value       = module.optimized-mysql-server-read-replica.db_username
+  sensitive   = true
+}
+
+output "replica_mysql_password" {
+  description = "MySQL instance master password."
+  value       = module.optimized-mysql-server-read-replica.db_password
+  sensitive   = true
+}
```
Comment (dillondoxey) [inline examples/intel-optimized-mysql-server-replica_testing/outputs.tf:103]: Do we want to output the password for this in cleartext by default?

File: examples/intel-optimized-mysql-server-replica_testing/main.tf [hunk 2]
```diff
@@ -5,23 +5,22 @@
...
   source         = "intel/aws-mysql/intel"
-  rds_identifier = "mysql-dev"
+  rds_identifier = "mysqldev"
   db_password    = var.db_password
 
   # Update the vpc_id below for the VPC that this module will use. Find the default vpc-id in your AWS account
   # from the AWS console or using CLI commands. In your AWS account, the vpc-id is represented as "vpc-",
   # followed by a set of alphanumeric characters. One sample representation of a vpc-id is vpc-0a6734z932p20c2m4
-  vpc_id = "<ENTER VPC_ID HERE>"
+  vpc_id = "vpc-0abf36f0171c55ae1"
```
Comment (dillondoxey) [inline examples/intel-optimized-mysql-server-replica_testing/main.tf:14]: Do we want to use `var.vpc_id` instead?

File: examples/intel-optimized-mysql-server-replica_testing/main.tf [hunk 3]
```diff
@@ -5,23 +5,22 @@
...
 module "optimized-mysql-server-read-replica" {
   source         = "intel/aws-mysql/intel"
-  rds_identifier = "mysql-dev-replica"
+  rds_identifier = "mysqldevreplica"
   db_password    = var.db_password
 
   # Update the vpc-id below. Use the same vpc-id as the one used in the prior module.
-  vpc_id                           = "<ENTER VPC_ID HERE>"
-  aws_database_instance_identifier = "mysql-rr"
+  vpc_id                           = "vpc-0abf36f0171c55ae1"
```
Comment (dillondoxey) [inline examples/intel-optimized-mysql-server-replica_testing/main.tf:23]: Do we want to use `var.vpc_id` instead?

**Review by dillondoxey — COMMENTED** [review]
Feedback provided. I think we should change the password output but I could go either way on using `var.vpc_id`

File: examples/intel-optimized-mysql-server-replica_testing/outputs.tf (hunk 1 above)
Comment (dillondoxey) [reply examples/intel-optimized-mysql-server-replica_testing/outputs.tf:103]: Too early! I missed the sensitive value. This is all set.

File: examples/intel-optimized-mysql-server-replica_testing/main.tf (hunk 2 above)
Comment (JoshHilliker) [reply examples/intel-optimized-mysql-server-replica_testing/main.tf:14]: why is the vpc_id in here?  the actual #..

File: examples/intel-optimized-mysql-server-replica_testing/main.tf (hunk 3 above)
Comment (JoshHilliker) [reply examples/intel-optimized-mysql-server-replica_testing/main.tf:23]: same on here..  vpc_id?

File: examples/intel-optimized-mysql-server-replica_testing/main.tf (hunk 2 above)
Comment (lucasmelogithub) [reply examples/intel-optimized-mysql-server-replica_testing/main.tf]: changed to <ENTER VPC_ID HERE> 
We'll make var.vpc_id in the future.

File: examples/intel-optimized-mysql-server-replica_testing/main.tf (hunk 3 above)
Comment (lucasmelogithub) [reply examples/intel-optimized-mysql-server-replica_testing/main.tf]: Changed to <ENTER VPC_ID HERE>
We'll make it a variable in the future.

**Review by JoshHilliker — APPROVED** [review]

**Pull request merged**